lib/QubitXmlImport.class.php

```diff
@@ -681,23 +695,33 @@ private function processMethods(&$domNode, &$importDOM, $methods, &$currentObjec
               $dacsProcessingInformationNoteTypeId   = array_search("Processing information", $termData['dacsSpecializedNotesTypes']['en']);
               $dacsVariantTitleInformationNoteTypeId   = array_search("Variant title information", $termData['dacsSpecializedNotesTypes']['en']);
 
-              // invoke the object and method defined in the schema map
-              $obj = call_user_func_array(array( & $currentObject, $methodMap['Method']), $parameters);
+              // Invoke the object and method defined in the schema map
+              $result = call_user_func_array(array( & $currentObject, $methodMap['Method']), $parameters);
 
               // If an actor/event object was returned, track that
               // in the events cache for later cleanup
-              if(!empty($obj))
+              if($currentObject instanceof QubitInformationObject && !empty($result))
```

Comment:
Minor nitpick, space here after if (I realize it was like this before too heheh)